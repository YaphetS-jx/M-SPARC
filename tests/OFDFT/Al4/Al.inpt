# Test: Al #
CELL: 8 8 8
MESH_SPACING: 0.5
BC: P P P
EXCHANGE_CORRELATION: LDA_PZ

TOL_PSEUDOCHARGE: 1.00E-08

OFDFT_FLAG: 1
TOL_OFDFT: 1E-3
OFDFT_LAMBDA: 0.2