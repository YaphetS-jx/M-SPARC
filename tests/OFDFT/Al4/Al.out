***************************************************************************
*                      M-SPARC v1.0.0 (Mar 29, 2021)                      *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 12-Apr-2021  Start time: 11:26:56                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 8.000000 8.000000 8.000000 
FD_GRID: 16 16 16
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
EXCHANGE_CORRELATION: LDA_PZ
OFDFT_FLAG: 1
OFDFT_LAMBDA: 0.200000
TOL_OFDFT: 1.00E-03
CALC_STRESS: 0
CALC_PRES: 0
TOL_PSEUDOCHARGE: 1.00E-08
REFERENCE_CUTOFF: 0.50
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/OFDFT/Al4/Al.out
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing                       :  0.500000 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  ../tests/OFDFT/Al4/Al.out
Total number of atom types         :  1
Total number of atoms              :  4
Total number of electrons          :  12
Atom type 1  (valence electrons)   :  Al 3
Pseudopotential                    :  Al_goodwin.psp8
lloc                               :  0
Pseudocharge radii of atom type 1  :  11.00 11.00 11.00
Number of atoms of type 1          :  4
Estimated total memory usage       :  12.94 MB
=====================================================================
                Orbital-Free DFT NLCG (OFDFT-NLCG#1)                
=====================================================================
Iteration      Free Energy (Ha/atom)   NLCG Error        Timing (sec)
1              -2.2114653075E+00       1.125E-02         0.225
2              -2.2123445996E+00       7.638E-03         0.016
3              -2.2130046001E+00       2.992E-03         0.013
4              -2.2130643674E+00       1.453E-03         0.024
5              -2.2130835219E+00       6.365E-04         0.014
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -2.2130835219E+00 (Ha/atom)
Total free energy                  : -8.8523340876E+00 (Ha)
Kinetic energy                     :  3.3051378145E+00 (Ha)
Exchange correlation energy        : -3.1457346917E+00 (Ha)
Self and correction energy         : -1.6403679287E+01 (Ha)
Average force                      :  1.2184663551E-15 (Ha/Bohr)
Maximum force                      :  1.7845997268E-15 (Ha/Bohr)
Time for force calculation         :  0.522 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  3.311 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
